File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex13.java
//Importando Scanner
import java.util.Scanner;

public class Ex13 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Médias\n\n");
		
		//vars
		double media, menorMedia, maiorMedia, mediaTurma=0, alunos=0;
		
		//Entrada
		System.out.print("Digite a média do aluno: ");
		media = ent.nextDouble();
		
		menorMedia = media;
		maiorMedia = media;
		
		while ( media >= 0)
		{
			//Processamento
			alunos++;
			mediaTurma += media;
			
			if( media < menorMedia )
				menorMedia = media;
			else if ( media > maiorMedia)
				maiorMedia = media;
			
			//Entrada
			System.out.print("Digite a média do aluno: ");
			media = ent.nextDouble();
				
		}
		
		mediaTurma /= alunos;
		
		//Saída
		System.out.println("Menor média: " + menorMedia);
		System.out.println("Maior média: " + maiorMedia);
		System.out.println("Média da turma: " + mediaTurma);
		
		
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex13.c
#include <stdio.h>

main()
{
    printf("Media de varios alunos e da turma\n\n");

    //vars
    float mediaTurma=0, mediaAluno, maiorMedia, menorMedia, alunos=0;

    //Entrada
    printf("\n\nInsira a media do aluno: ");
    scanf("%f", &mediaAluno);

    while( mediaAluno >= 0 )
    {
        alunos++;

        mediaTurma += mediaAluno;

        //Entrada
        printf("\n\nInsira a media do aluno: ");
        scanf("%f", &mediaAluno);


    }

    //Processamento
    mediaTurma /= alunos;

    //Saída
    printf("\n\nMedia da Turma: %.2f", mediaTurma);




}
